packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -461,8 +461,8 @@ export class AotPlugin implements Tapable {
             if (this.skipCodeGeneration) {
               this._lazyRoutes[k] = lazyRoute;
             } else {
-              const lr = path.relative(this.basePath, lazyRoute.replace(/\.ts$/, '.ngfactory.ts'));
-              this._lazyRoutes[k + '.ngfactory'] = path.join(this.genDir, lr);
+              const lr = path.relative(this.basePath, lazyRoute.replace(/(\.d)?\.ts$/, '.ngfactory.ts'));
+              this._lazyRoutes[k + '.ngfactory'] = path.join(this.genDir, path.join('/', lr));
```

Comment:
Thanks for the comment! I'm thinking I should have used path.sep instead to be platform agnostic. Will update the PR shortly.